FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_DNS.c

```diff
@@ -1136,14 +1202,21 @@ BaseType_t xDoStore = xExpected;
 					}
 
 					pucByte += sizeof( DNSAnswerRecord_t ) + sizeof( uint32_t );
-					uxSourceBytesRemaining -= ( sizeof( DNSAnswerRecord_t ) + sizeof( uint32_t ) );
+					/* Commenting this out since this is not being used later anywhere. 
+					 * Being cautious by not removing it completely
+					 * uxSourceBytesRemaining -= ( sizeof( DNSAnswerRecord_t ) + sizeof( uint32_t ) );
```

Comment:
I don't understand the rationale of commenting it out but leaving it in for caution. The cautious approach would be to not change the code unless there's a definite bug.

Reply:
I removed it completely. It is a local variable and it is not being used after this statement. Furthermore, there is a break after this statement which will ensure that the variable is not used in the loop.
Would you like me to leave it in and allow compiler to remove it later?